packages/reports/src/Containers/open-positions.tsx

```diff
@@ -190,23 +191,23 @@ const OpenPositions = observer(({ component_icon, ...props }: TOpenPositions) =>
     const { isDesktop } = useDevice();
     const previous_active_positions = usePrevious(active_positions);
     const contract_types = [
-        { text: localize('Options'), is_default: !is_multiplier && !is_accumulator },
-        { text: localize('Multipliers'), is_default: is_multiplier },
-        { text: localize('Accumulators'), is_default: is_accumulator },
+        { text: 'Options', is_default: !is_multiplier && !is_accumulator },
```

Comment:
Why do you remove localize here? Am I missing something?

Reply:
This localize caused a bug. We shouldn't wrapped with localize both text and value